pkg/codegen/internal/test/testdata/functions.pp.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Pulumi;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        var encoded = Convert.ToBase64String(System.Text.UTF8.GetBytes("haha business"));
+        var joined = string.Join("-", 
+        {
+            "haha",
+            "business",
+        });
```

Comment:
One last thing for posterity... I don't it's related to this change specifically, but this doesn't compile.

It'd need to be something like:

```suggestion
        var joined = string.Join("-", new[]
        {
            "haha",
            "business",
        });
```

Or:

```suggestion
        var joined = string.Join("-", new string[]
        {
            "haha",
            "business",
        });
```

Or, since the last argument is a params array, in this specific case it could be:

```suggestion
        var joined = string.Join("-", "haha", "business");
```